### PR TITLE
Improve tutorials for Job

### PR DESCRIPTION
**Pull request by sftim**
I've made some changes to some of the tutorials for Job:
- https://kubernetes.io/docs/tasks/job/coarse-parallel-processing-work-queue/
- https://kubernetes.io/docs/tasks/job/fine-parallel-processing-work-queue/

Now, the tutorials match our style guide a lot better.

There's a lot more to do! Ideally we'd redo them from the ground up.

However, I hope these look like a positive set of changes to make, for now.

### CODE REVIEW

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 1]
```diff
@@ -77,76 +80,82 @@ Waiting for pod default/temp-loe07 to be running, status is Pending, pod ready:
...
+```
 ```
-# env | grep RABBIT | grep HOST
 RABBITMQ_SERVICE_SERVICE_HOST=10.0.147.152
-# Your address will vary.
 ```
+(the IP address will vary)
 
-Next we will verify we can create a queue, and publish and consume messages.
+Next you will verify that can create a queue, and publish and consume messages.
```
Comment (kundan2707) [inline content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md:120]:
```suggestion
Next you will verify that you can create a queue, and publish and consume messages.
```

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+In this example, you will run a Kubernetes Job with multiple parallel
 worker processes.
 
 In this example, as each pod is created, it picks up one unit of work
 from a task queue, completes it, deletes it from the queue, and exits.
 
 Here is an overview of the steps in this example:
 
-1. **Start a message queue service.**  In this example, we use RabbitMQ, but you could use another
+1. **Start a message queue service.**  In this example, you use RabbitMQ, but you could use another
```
Comment (kbhawkey) [inline content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md:18]: nit: Could you instead say, The example uses RabbitMQ, but you could choose <another messaging broker>.

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 3]
```diff
@@ -157,33 +166,37 @@ In a practice, the content of the messages might be:
...
 
-## Create an Image
+## Create a container image
 
-Now we are ready to create an image that we will run as a job.
+Now you are ready to create an image that you will run as a Job.
 
-We will use the `amqp-consume` utility to read the message
-from the queue and run our actual program.  Here is a very simple
+The job will use the `amqp-consume` utility to read the message
```
Comment (kbhawkey) [inline content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md:198]: nit: Should you capitalize Job?

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md:18]: Yep, that also works.

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md (hunk 3 above)
Comment (kbhawkey) [reply content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md:198]: OK. I'd prefer consistent formatting and style of the word on the page.

**Pull request merged**